pip/requirements.txt

```diff
@@ -214,16 +216,16 @@ notebook-shim==0.2.2
 numexpr==2.8.4
 # setuptools version <64 is needed by numpy: https://github.com/pypa/setuptools/issues/3549
 # NO_AUTO_UPDATE: update together with tensorflow
-numpy==1.23.5
+numpy==1.24.3
```

Comment:
does this work with current version of TF?

Reply:
With 2.6.4? IIRC, no.